src/wp-includes/rest-api/search/class-wp-rest-post-search-handler.php

```diff
@@ -84,6 +84,9 @@ public function search_items( WP_REST_Request $request ) {
 
 		$query     = new WP_Query();
 		$found_ids = $query->query( $query_args );
+		if ( ! empty( $found_ids ) ) {
```

Comment:
Another solution might be to do this.

```
$query_args = array(
			'post_type'           => $post_types,
			'post_status'         => 'publish',
			'paged'               => (int) $request['page'],
			'posts_per_page'      => (int) $request['per_page'],
			'ignore_sticky_posts' => true,
			'update_post_term_cache' => false, 
                        'update_post_meta_cache' => false,
		);

$query     = new WP_Query();
$posts = $query->query( $query_args );
$found_ids = wp_list_pluck( $posts, 'ID' );
```

The benefit of this, it more likely to go through caching plugins.